### PR TITLE
feat dataManipulation

### PR DESCRIPTION
**Pull request by Fav8**
this will implement the dataManipulation function inside the SentimentModel and the faceAnalysis interface

### CODE REVIEW

File: src/models/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { IFaceDetails } from './interface';
```
Comment (tinazhouhui) [inline src/models/helpers.ts:1]: quick question, do we need this file at all? I feel like removeuseless props will probably live in rekkognition model and calculate the scores should live inside the sentiment model no?

**Review by tinazhouhui — CHANGES_REQUESTED** [review]

File: src/models/helpers.ts (hunk 1 above)
Comment (tinazhouhui) [reply src/models/helpers.ts:1]: up to you :) technically speaking the role of the sentiment model is exactly what you just wrote tho - the model that takes care of the data :)

**Comment by Fav8** [comment]
EDIT: implementation of th efull datamanipulation flow, including the mock request to aws reckon.
only thing missing is the isImportant attribute to each frame in the frameinfo array because I need to talk with filo and the rest of you guys about it.

File: src/models/RekognitionModel.ts [hunk 2]
```diff
@@ -11,8 +12,8 @@ export default class RekognitionModel extends BaseModel {
     this.rekognition = rekognitionConnection.getClient();
   }
 
-  detectFaces (imageIds: string[]) {
-    return;
+  detectFaces (image: string) {
+    return mockResponse;
```
Comment (tinazhouhui) [inline src/models/RekognitionModel.ts:16]: just to make sure that I understand, whatever image we pass in we get the first mock response, correct?

**Comment by Fav8** [comment]
merged, closing

**Pull request merged**